python_modules/libraries/dagster-embedded-elt/dagster_embedded_elt_tests/dlt_tests/conftest.py

```diff
@@ -0,0 +1,28 @@
+import os
+from typing import Generator, Tuple
+
+import dlt
+import pytest
+from dlt import Pipeline
+from dlt.extract.source import DltSource
+
+from .constants import EXAMPLE_PIPELINE_DUCKDB
+from .dlt_test_sources.duckdb_with_transformer import pipeline
+
+
+@pytest.fixture(autouse=True, scope="module")
+def setup_dlt_pipeline() -> Generator[Tuple[DltSource, Pipeline], None, None]:
+    if os.path.exists(EXAMPLE_PIPELINE_DUCKDB):
+        os.remove(EXAMPLE_PIPELINE_DUCKDB)
+
+    dlt_source = pipeline()
```

Comment:
This is static, so I don't think this should be yielded by a fixture. Just instantiate it in the top-level of your test file.